### ISSUE TITLE
`block-no-empty` rule doesn't work with SugarSS

### ISSUE THREAD

**Issue by FezVrasta** [stylelint/stylelint#2272]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where I define an empty block with SugarSS, the linter doesn't throw an error.

> Which rule, if any, is this issue related to?

`block-no-empty`

> What CSS is needed to reproduce this issue?

e.g.
```sugarss
.Foo
  color: red
  &--bar
  background-color: yellow // not the missing indentation
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "block-no-empty": true
  }
}
```

> Which version of stylelint are you using?

7.7.1

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI `stylelint .`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes it is related to nesting and SugarSS

> What did you expect to happen?

A warning to be flagged

> What actually happened (e.g. what warnings or errors you are getting)?

No warnings
